### DISCUSSION TITLE
A colab for this would be godsend!

### DISCUSSION THREAD

**Discussion by gilroff** [bmaltais/kohya_ss#368] [Ideas]
Like many people I don't have a good gpu :(

**Comment by Norian11** [comment]
Well there are many collabs that use the scripts of Kohya, the most used is the one from here https://github.com/Linaqruf/kohya-trainer
I normally select Kohya Lora Dreambooth because the Fast Trainer one in the past was sending me error, there are many guides on how to use it in YouTube

**Reply by gilroff** [reply]
thankss
